paper/paper.Rmd

```diff
@@ -217,11 +219,13 @@ There are two main ways to customize it:
 
   - Create new linters (by leveraging functions like `lintr::make_linter_from_xpath()`) tailored to match project- or organization-specific coding standards.
 
+Indeed, `{goodpractice}` [@Padgham2024] bundles a set of custom linters that are not part of the default set of `{lintr}` linters, while `{flint}` [@Bacher2024] provides a way to fix the lints detected by `{lintr}`. Moreover, `{box.linters}` [@Basa2024] extends `{lintr}` to support `{box}` modules [@Rudolph2024], while `{checklist}` includes linters as one of the strict checks for R packages [@Onkelinx2024].
```

Comment:
Hmm, not strictly accurate. {flint} does not user {lintr}'s code -- the backends are not related. But the {flint} author has copied most of the rules we suggest & translated them to work on the {flint} backend in Rust {astgrepr}.

Reply:
Any suggestion on how to do justice to `{flint}`? 
The reviewer has explicitly mentioned it in their feedback, and so we do need to reference it.